application/protected/components/ApiAxle/Client.php

```diff
@@ -135,13 +135,18 @@ public function getApiInfo($apiName)
                 $this->getDataFromResponse($response)
             );
         } catch (\Exception $e) {
-            if ($e->getCode() === 404) {
+            if (self::got404($e)) {
                 return null;
             }
             $this->throwWithBetterMessage($e);
         }
     }
-    
+
+    private function got404(\Exception $e) {
+        return ($e->getCode() === 404 ||
+            str_contains($e->getMessage(), "resulted in a `404 Not Found`"));
+    }
```

Comment:
It looks like this is being used statically, so it should probably be made a `static` function:
```suggestion
    private static function got404(\Exception $e)
    {
        return ($e->getCode() === 404 ||
            str_contains($e->getMessage(), "resulted in a `404 Not Found`"));
    }
```